app/html/responder_profile.html

```diff
@@ -188,25 +190,34 @@ <h3>Settings</h3>
     <!-- DASHBOARD CONTENT -->
     <div class="main">
       <section class="profile tabs">
-        <h1>Your Posts</h1>
-        <div class="userpost">
-          <div class="posts">
-            <span><i class="material-icons">add</i></span>
-          </div>
-          <div class="posts">
-            <img width="200" src="/imgs/saveme.jpg" alt="">
-            <p>Title</p>
-          </div>
-          <div class="posts">
-            <img width="200" src="/imgs/saveme.jpg" alt="">
-            <p>Title</p>
-          </div>
-          <div class="posts">
-            <img width="200" src="/imgs/saveme.jpg" alt="">
-            <p>Title</p>
-          </div>
+        <h1>User profile</h1>
+        <div id="user_profile_section">
+          <img width="200" src="/imgs/saveme.jpg" alt="">
+          <p>First Name: </p>
+          <div class="edit" id="detail_user_firstN" >bank</div>
+          <p>Last Name: </p>
+          <div class="edit" id="detail_user_lastN" >name</div>
+          <p>Email: </p>
+          <div class="edit" id="detail_user_email" >email</div>
+          <p>Password: </p>
+          <div class="edit" id="detail_user_password" >name</div>
+          <p>Age: </p>
+          <div class="edit" id="detail_user_age" >age</div>
+          <p>Gender: </p>
+          <div class="edit" id="detail_user_gender" >gender</div>
+          <p>Phone Number: </p>
+          <div class="edit" id="detail_user_cellphone">number</div>
+          <p>Role: </p>
+          <div class="edit" id="detail_user_role">role</div>
+          <br><br>
```

Comment:
responder profile page doesn't show editing errors?

Reply:
Right now, I have responder_profile.html validated the user input.